tests/WebUI.IntegrationTests/IntegrationTestHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Codidact.WebUI.IntegrationTests
+{
+    public static class IntegrationTestHelper
+    {
+        public static StringContent GetRequestContent(object obj)
+        {
+            return new StringContent(JsonConvert.SerializeObject(obj), Encoding.UTF8, "application/json");
+        }
+
+        public static async Task<T> GetResponseContent<T>(HttpResponseMessage response)
+        {
+            var stringResponse = await response.Content.ReadAsStringAsync();
+
+            var result = JsonConvert.DeserializeObject<T>(stringResponse);
```

Comment:
~~~diff
--- a/tests/WebUI.IntegrationTests/IntegrationTestHelper.cs
+++ b/tests/WebUI.IntegrationTests/IntegrationTestHelper.cs
@@ -16,9 +16,7 @@ namespace Codidact.WebUI.IntegrationTests
         {
             var stringResponse = await response.Content.ReadAsStringAsync();
 
-            var result = JsonConvert.DeserializeObject<T>(stringResponse);
-
-            return result;
+            return JsonConvert.DeserializeObject<T>(stringResponse);
         }
     }
 }
~~~

Reply:
No, this is fine. Its easier to debug if its split in to two lines.

Reply:
^ as asynts says, this can be a straight `return` without assignment.

Reply:
Well, majority rule. Alright, will do